README.md

```diff
@@ -50,11 +50,30 @@ Copy the `dist/omg.js` file into your project and include it.
 
 ### Examples
 
-You can find example code inside `examples` folder
+You can find example code inside `examples` folder. To run the examples, step inside the `examples` directory. Run `npm install`. Edit your `config.js` file to have to correct values.
 
-[Deposit ETH through Rootchain Contract](examples/node-deposit.js)
+_Note: You may need to also run `npm install` in the `packages\omg-js-rootchain`, `packages\omg-js-childchain` and `packages\omg-js-util` directories._
 
-[Transact ETH on the Child chain](examples/node-transact.js)
+Let's run through a story between Alice and Bob. In this story, Alice will first deposit 2 ETH from the root chain into the child chain. Then Alice will transfer 1 ETH to Bob on the child chain. Finally, Bob will exit his funds from the child chain back into the root chain. His root chain balance will be reflected with the extra 1 ETH that Alice sent to him on the child chain.
+
+From the `examples` folder run the scripts below in order to follow the story.
+
+1. [Deposit 2 ETH from Alice's Rootchain to the Childchain](examples/childchain-deposit.js)
```

Comment:
I'm not sure if we want to hard code this ? a user may have less than 2 ETH, probably better to have this value be configurable

Reply:
Agreed. Will change this.